src/storage/query/QueryBaseProcessor.inl

```diff
@@ -456,6 +513,16 @@ kvstore::ResultCode QueryBaseProcessor<REQ, RESP>::collectEdgeProps(
                                                   val,
                                                   spaceId_,
                                                   std::abs(edgeType));
+            // Check if the schema has TTL
+            auto retTTL = getEdgeTTLInfo(edgeType);
```

Comment:
Move it outside of the loop

Reply:
The comment is incomplete here， I will  fix
Here check if each row of data is expired,  so get the value of each row of data to check。
So it does not move it outside of the loop

Reply:
why call "getEdgeTTLInfo" for every edge?  The same edgeType has the same TTL Info.

Reply:
You are right.
My mistake.

👍👍👍,

Reply:
I mistakenly thought  move "checkDataExpiredForTTL" . My mistake.